src/AcmHeader/AcmHeader.tsx

```diff
@@ -335,6 +335,7 @@ export enum AcmRoute {
     Applications = '/multicloud/applications',
     Credentials = '/multicloud/credentials',
     Governance = '/multicloud/policies',
+    InfraEnvironments = '/multicloud/infra-environments',
```

Comment:
Do we need to mark AcmRoute.InfraEnvironments as a console route at line 370?

Reply:
Right, I missed that.
Code is now updated.